src/bootstrap/dist.rs

```diff
@@ -98,6 +98,139 @@ pub fn docs(build: &Build, stage: u32, host: &str) {
     }
 }
 
+fn find_files(files: Vec<&str>, path: &Vec<PathBuf>) -> Vec<PathBuf> {
```

Comment:
I'd prefer that this function take `files: &[Path], path: &[PathBuf]` since it doesn't need a vector for either parameter. Looks like the files are really just strings, in which case `&[str]` would also be fine.